sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/ScriptTransformationSuite.scala

```diff
@@ -75,49 +53,4 @@ class ScriptTransformationSuite extends SparkPlanTest {
       )(TestHive),
       rowsDf.collect())
   }
-
-  test("script transformation should not swallow errors from upstream operators (no serde)") {
-    val rowsDf = Seq("a", "b", "c").map(Tuple1.apply).toDF("a")
-    val e = intercept[TestFailedException] {
-      checkAnswer(
-        rowsDf,
-        (child: SparkPlan) => new ScriptTransformation(
-          input = Seq(rowsDf.col("a").expr),
-          script = "cat",
-          output = Seq(AttributeReference("a", StringType)()),
-          child = ExceptionInjectingOperator(child),
-          ioschema = noSerdeIOSchema
-        )(TestHive),
-        rowsDf.collect())
-    }
-    assert(e.getMessage().contains("intentional exception"))
-  }
-
-  test("script transformation should not swallow errors from upstream operators (with serde)") {
```

Comment:
Maybe I need to add this back, but seems like if the `child` throw exception, then the actual result should be null which would cause `checkAnswer` throw not equal exception first instead of "intentional exception"